From 2d5bd304209b56e89d454037c330e26de2c5f970 Mon Sep 17 00:00:00 2001
From: Jiansong Chen <Jiansong.Chen@amd.com>
Date: Thu, 19 Dec 2019 14:58:42 +0800
Subject: [PATCH 1129/1453] drm/amdkcl: fix implicit declaration of
 prepare_flip_isr

dm_page_flip is activated for CentOS7.3 kernel(drm version
4.6.5). The funciton will call prepare_flip_isr and its
definition is in front of prepare_flip_isr.

Change-Id: I084653267001f327adb75f05b9b8f6608af1d9c1
Signed-off-by: Jiansong Chen <Jiansong.Chen@amd.com>
Reviewed-by: Flora Cui <flora.cui@amd.com>
Reviewed-by: Jack Gui <Jack.Gui@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index d1ecaf9bfe3c..bc243c9f9565 100755
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -163,6 +163,8 @@ static int amdgpu_dm_atomic_check(struct drm_device *dev,
 static void handle_cursor_update(struct drm_plane *plane,
 				 struct drm_plane_state *old_plane_state);
 
+static void prepare_flip_isr(struct amdgpu_crtc *acrtc);
+
 static void amdgpu_dm_set_psr_caps(struct dc_link *link);
 static bool amdgpu_dm_psr_enable(struct dc_stream_state *stream);
 static bool amdgpu_dm_link_setup_psr(struct dc_stream_state *stream);
-- 
2.17.1

